app/src/userpages/components/ProfilePage/index.jsx

```diff
@@ -102,41 +100,39 @@ export class ProfilePage extends Component<Props, State> {
                 <MediaQuery minWidth={lg.min}>
                     <ConfigureAnchorOffset value={-80} />
                 </MediaQuery>
-                <DetailsContainer className={styles.profilePage}>
```

Comment:
Leaving out the container might cause the "Field editor protrudes outside its container" issue since that container limits inputs with the class `.constrainInputWidth`.